src/components/Form/Form.tsx

```diff
@@ -135,22 +135,22 @@ const Form = <T extends FormDetails>({
               onClick={props.onCancel}
               label={props.cancelLabel ?? "Cancel"}
             />
-            <Submit
-              id="save"
-              data-testid="submit-button"
-              type="submit"
-              disabled={disabled || !isDirty}
-              label={props.submitLabel ?? "Submit"}
-            />
             {props.showSaveAndAddMoreBtn && (
               <Submit
-                id="save-and-add-more"
+                id="save"
```

Comment:
🤔 This should have the `save-and-add-more` as Id, and button below should correspond to regular save button.

Please make sure to test/check with what's on develop branch (or care.ohc.network).